Makefile.in

```diff
@@ -1,7 +1,7 @@
-# Makefile.in generated by automake 1.16.1 from Makefile.am.
+# Makefile.in generated by automake 1.15.1 from Makefile.am.
```

Comment:
you regressed the version of automake. is that a problem? guessing not.

Reply:
Ya moving back and forth between Ubuntu 18.04 and Fedora 32.

Reply:
I clean this up with the Makefile.am pull request.